Seqz preprocess@0.2.0

lDesiree opened the pull request:
[release]

lDesiree committed:
[wfpm v0.7.8] added starting template for seqz-preprocess@0.2.0

lDesiree committed:
[release]

lDesiree committed:
change name of output file

lDesiree committed:
changed sqz-preprocess output filename

lDesiree committed:
[release]

lDesiree committed:
[release]

junjun-zhang commented on snp-pileup/tests/test-job-1.json:
```diff
@@ -3,7 +3,7 @@
     "dbsnp": "input/test.dbsnp_138.hg19.vcf",
     "expected_output": "expected/expected.bc.gz",
-    "publish_dir": "outdir",
+    "publish_dir": ".",
```
I guess this is not intended? typically, when working on one package, we don't make change to files do not belong to the package. WFPM CLI currently does not enforce this rule, but I think it would be useful to implement it in WFPM CLI

lDesiree replied:
yes it wasn't intended - will change it back and push again.

junjun-zhang commented on seqz-preprocess/main.py:
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```
this main.py file is generated at the time the package was created, it is indeed completely optional. If not used, it can be removed.

lDesiree replied:
thanks I will remove it

junjun-zhang committed:
add test job

junjun-zhang commented on nextflow.config:
```diff
@@ -1,10 +0,0 @@
-docker {
-    enabled = true
-    runOptions = '-u \$(id -u):\$(id -g)'
-}
```
this file is not part of the package, should not be updated in this branch

lDesiree committed:
reverted a couple of unintended changes

lDesiree committed:
merge
Merge branch 'seqz-preprocess@0.2.0' of https://github.com/ICGC-ARGO-Structural-Variation-CN-WG/wfpm-demo into seqz-preprocess@0.2.0

junjun-zhang reviewed (approved):
looking good
